.evergreen/failing-tests

```diff
@@ -0,0 +1,67 @@
+# This file lists known failing tests that for some reason or another cannot be
+# fixed and rather should be skipped in Evergreen builds.  Note that listing a
+# test in this file will result in the test being skipped on all platforms,
+# build variants, tasks, etc.
+#
+# The format of this file is as follows:
+# - one test per line, with an optional comment (total lines in file < 1,000)
+# - comments begin with a hash (#) character and continue to the end of the line
+# - lines starting with the comment character and blank lines are ignored
+# - specify test names as they are known to the test runner (e.g., /some/test)
+# - specify sub-tests of spec tests by an additional slash (/) and the quoted description
+#   (e.g., /some/spec/test/"sub-test 1")
+# - placing a comment at the end of a line will cause the test runner to ouptut
+#   the comment text as the skip reason when the test is skipped
```

Comment:
OK.  Thanks for the suggestion.  I think I can accomplish what you are suggesting by simply making this change:

```
-      if (comment_char) {
+      if (comment_char && comment_char[1] == '#') {
```

Since the string is NULL-terminated, I think that in the worst case (the '#' is at the end of the string), the `comment_char[1]` is still OK, since in that case it would just encounter the `\0`.  What I'm trying to say is that I think this simple approach would never result in reading past the end of the string.  Do you agree?  Or did I miss something in how I thought about it?

Reply:
I agree that it's not a /bug/-- but re-using the same character for two very different meanings is surprising.

Reply:
Thinking about this some more, it doesn't seem like re-using the same character in two different contexts.  The `#` character signifies a comment.  When the comment is on the same line as a test to skip, then the comment is output during test execution to signify why that particular test was skipped.  I'll re-read the documentation at the start of the skip file and see if there is a way to explain with more clarity.

Reply:
As long as you've thought it through and are ok with it I'm sure that what you choose will be fine.